hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/RegionFactoryDefaultSlowTest.java

```diff
@@ -61,4 +65,19 @@ public void testQueryCacheCleanup() {
         assertEquals(numberOfEntities - evictedItemCount, queryRegion.getCache().size());
     }
 
+    @Test
+    public void testUpdate() {
+        insertDummyEntities(1);
+        Session session = sf.openSession();
+        Transaction tx = session.beginTransaction();
+        DummyEntity ent = (DummyEntity) session.get(DummyEntity.class, 0L);
```

Comment:
The test is a bit obscure because they key '0' appears out of thin air. I guess you know there is an entry created with key 0 when insertDummyEntities(1) is being called.